rpc-protocol/src/main/java/org/sonarsource/sonarlint/core/rpc/protocol/backend/tracking/TrackWithServerIssuesParams.java

```diff
@@ -23,6 +23,7 @@
 import java.util.List;
 import java.util.Map;
 
+@Deprecated(since = "10.2")
```

Comment:
It just came to my mind that there is a similar method for hotspots: `org.sonarsource.sonarlint.core.rpc.protocol.backend.tracking.SecurityHotspotMatchingRpcService#matchWithServerSecurityHotspots`. It can probably be deprecated too. And by the way we might need a test for hotspots too

Reply:
Tracked with SLCORE-788